cloudwatch/cloudwatch.js

```diff
@@ -22,7 +22,20 @@ class CloudwatchWrapper {
     return result;
   }
 
-  sendCustomCountMetric(namespace, metricName, value){
+  putMetricAlarm(alarmParams) {
+    
+    const result = this.cloudwatch.putMetricAlarm.promise()
+      .then(data => {
+        logger.debug(`Successful alarm ${data} send to CloudWatch: ${JSON.stringify(alarmParams)}`)
+      })
+      .catch(err => {
```

Comment:
Need to implement checks for known `Error` classes and dispatch known errors with `CommonError` being the default if no more specific errors are received.

See here for reference:
https://github.com/sudokrew/sudokrew-aws-sdk/blob/53275914eb975ca170071520a09c40bb5018a319/sqs/sqs.js#L31